src/mapping/AxialGeoMultiscaleMapping.cpp

```diff
@@ -0,0 +1,143 @@
+#include "AxialGeoMultiscaleMapping.hpp"
+#include "mesh/Mesh.hpp"
+
+namespace precice {
+
+namespace mapping {
+
+AxialGeoMultiscaleMapping:: AxialGeoMultiscaleMapping
+(
+  Constraint constraint,
+  int        dimensions,
+  MultiscaleType type,
+  double radius)
+:
+  Mapping(constraint, dimensions),
+  _type(type),
+  _radius(radius)
+{
+  setInputRequirement(Mapping::MeshRequirement::VERTEX);
+  setOutputRequirement(Mapping::MeshRequirement::VERTEX);
+
+  PRECICE_ASSERT(dimensions == 2 || dimensions == 3);
+  if(dimensions == 2){
+    _scaling = 1.5;
+  } else if(dimensions == 3){
+    _scaling = 2.0;
+  }
```

Comment:
The `_scaling` being different for 2D and 3D cases is something that I'm not sure is correct. In my opinion it should be `2.0` no matter the dimensionality. We should keep track of it and investigate it further. However, for now we're only dealing with 3D cases so it can stay.

Reply:
Let's document the "maybe this is not correct" part. Maybe `_scaling` is just a confusing name in the end.